en/docs/authenticate/register-an-sp.md

```diff
@@ -4,4 +4,4 @@
 
```

Comment:
Do we have any special reason to have the page name as 'register-an-sp'?

Reply:
Not really... Didn't put much thought into the page name since it's an included page and the title won't appear anywhere. Shall I rename this to "register-a-service-provider.md"?